scripts/mktcli.js

```diff
@@ -48,23 +59,47 @@ const main = {
       )
 
     program
-      .command('land-owner <coord>')
-      .description('Get the land owner of a (x,y) coordinate')
+      .command('asset-owner <assetId> <assetType>')
+      .description('Get the owner of an asset')
       .action(
-        asSafeAction(async coord => {
-          const [x, y] = parseCLICoords(coord)
-          const contract = eth.getContract('LANDRegistry')
-          const owner = await contract.ownerOfLand(x, y)
+        asSafeAction(async (assetId, assetType) => {
+          const asset = await getAssetFromCLIArgs(assetId, assetType)
+          let contract
+
+          switch (assetType) {
```

Comment:
What do you think of moving this to a helper function to retrieve the contract based on the `assetType`? we can use it in other places

Reply:
I like it, but where would you define it?

Reply:
As it is an isolated file I do it here